### PR TITLE
Updates user status on task update

### PR DESCRIPTION
**Pull request by Ajeyakrishna-k**
## Description: 
When the task request is approved successfully and task is assigned to the user. User status state is updated to ACTIVE.

Date: 27 Oct 2023
Developer Name: @Ajeyakrishna-k 

Issue Ticket Number:- - #1583 

Backend changes
Yes

Frontend Changes
No

Is Under Feature Flag 
Yes

Database changes
No

Breaking changes (If your feature is breaking/missing something please mention pending tickets)
No

Deployment notes
N/A

Test code coverage: 
Screenshot 2023-10-27 at 3 30 43 AM

### CODE REVIEW

**Review by shubhamsigdar1 — COMMENTED** [review]
Hi @Ajeyakrishna-k, When user change task status in-progress from my-site then only 
user-status will be active.
@heyrandhir  did this change previously
you discussed above changes with @ankushdharkar???

File: controllers/tasksRequests.js [hunk 1]
```diff
@@ -167,6 +168,8 @@ const approveTaskRequest = async (req, res) => {
     if (response.isTaskRequestInvalid) {
       return res.boom.badRequest("Task request was previously approved or rejected.");
     }
+    await updateUserStatusOnTaskUpdate(user.username);
```
Comment (shubhamsigdar1) [inline controllers/tasksRequests.js:171]: Is this function update user-status to active??

Comment (Ajeyakrishna-k) [reply controllers/tasksRequests.js:171]: yes

**Comment by Ajeyakrishna-k** [comment]
> Hi @Ajeyakrishna-k, When user change task status in-progress from my-site then only user-status will be active. @heyrandhir did this change previously you discussed above changes with @ankushdharkar???
> 
> I read description then I am writing above...please correct me if I am wrong

Verified it with @ankushdharkar its needed.

**Review by heyrandhir — APPROVED** [review]

**Review by ardourApeX — APPROVED** [review]

**Review by Pratiyushkumar — APPROVED** [review]

**Pull request merged**